Update TMAP8 website (#96)
idaholab/TMAP8: doc/content/getting_started/installation.md

In order to install TMAP8, the MOOSE developer environment must be installed. The
installation procedure depends on your operating system, so click on the MOOSE
website link below that corresponds to your operation system/platform and follow

> the instructions until you are done with the step named "Install MOOSE" (note that you do not need to follow the steps from the section titled "Cloning MOOSE" and below). Then,

Reviewer: ```suggestion
the instructions until you are done with the step named "Install MOOSE" (note that 
you do not need to follow the steps from the section titled "Cloning MOOSE" and below). Then,
```

Marking a line-length/formatting change to also let you know one that GitHub wouldn't let me mark. Could you fix "operation system" on the line above and make it "operating system" while we're here? Thanks!

In order to install TMAP8, the MOOSE developer environment must be installed. The
installation procedure depends on your operating system, so click on the MOOSE

> website link below that corresponds to your operation system/platform and follow

Reviewer: ```suggestion
website link below that corresponds to your operating system/platform and follow
```

This was not fixed from my previous review, but I figured out how to suggest it this time. :)

```

This assumes that your MOOSE-based development environment is named `moose`. To update your TMAP8
repository as a TMAP8 user, use the following commands, which provide general users the most stable
branch (`upstream/main`):

```bash

> cd ~/projects/TMAP8
> git fetch upstream

Reviewer: ```suggestion
cd ~/projects/TMAP8
git checkout main
git fetch upstream
```

Let's also start requesting a checkout here to make sure they are updating against the right local branch.

git rebase upstream/main
```

> To update your TMAP8 repository as a TMAP8 developer who changes the code, use the following commands,

Reviewer: ```suggestion
To update your TMAP8 repository as a TMAP8 developer who regularly makes modifications to the code, use the following commands,
```

which provide developers with the `devel` branch:

```bash

> cd ~/projects/TMAP8
> git fetch upstream

Reviewer: ```suggestion
cd ~/projects/TMAP8
git checkout devel
git fetch upstream
```

```bash
cd ~/projects/TMAP8
git fetch upstream

> git rebase upstream/devel

Reviewer: We put `main` in the original version of these instructions because we wanted general users to use the most stable branch instead of devel, which should be for developers. Perhaps we need to be clearer here about which update is recommended for each group?

Author: I just updated these instructions to provide both, with clarification on who it applies to. Let me know if that looks good to you.

Reviewer: Made some follow-up comments, but I think we're good after this round. Thanks!
